pkg/publisher/publisher.go

```diff
@@ -90,8 +99,25 @@ type cloudWatchPublisher struct {
 
 // New returns a new instance of `Publisher`
 func New(ctx context.Context) (Publisher, error) {
+
+	httpTimeoutEnvInput := os.Getenv(httpTimeoutEnv)
```

Comment:
same as above comment

Reply:
Responded in other similar comment on other file